tests/cpu_bugs/kvm_guest_migration.pm

```diff
@@ -112,15 +113,21 @@ sub run {
     }
     elsif (get_var("MIGRATION_DEST")) {
 
-        #access this machine without password
-        assert_script_run("cp /etc/libvirt/libvirtd.conf{,.bak}");
-        assert_script_run(
-            "sed -i 's/#listen_tcp = 1/listen_tcp = 1/g' /etc/libvirt/libvirtd.conf"
-        );
-        assert_script_run(
-            "sed -i 's/#auth_tcp = .*/auth_tcp = \"none\"/g' /etc/libvirt/libvirtd.conf"
-        );
-        systemctl("restart libvirtd");
+        if (is_sle('>=15-sp2')) {
+            systemctl("start libvirtd-tcp.socket");
+        } else {
+            #access this machine without password
+
+            assert_script_run("cp /etc/libvirt/libvirtd.conf{,.bak}");
```

Comment:
just keep in mind that you running this test in qcow image which will be anyway destroyed after test will finished , also I fail to find code which revert saved to `.bak` so what the point ? I would just drop this line

Reply:
This case running on the physical machine by design....but I don't know if it could run in GUEST, because this is "install guest",  might be can't running in "guest"